test/Grid.spec.js

```diff
@@ -298,6 +299,26 @@ describe('<Grid />', () => {
       expect(rowFooter).to.have.lengthOf(1);
     });
   });
+  
+  describe('RemoteDataSource', () => {
+    it('should response with status 200 when column structure is OK', () => {
+      mock.onPost('http://tubular.azurewebsites.net/api/orders/paged', { columns }).reply(200);
+
+      axiosInstance.post('http://tubular.azurewebsites.net/api/orders/paged', { columns })
+        .then(response => {
+          expect(response.status).to.equal(200);
+        });
+    });
+
+    it('should response with status 500 when column structure is not OK', () => {
+      mock.onPost('http://tubular.azurewebsites.net/api/orders/paged', { fakeColumnStructure }).reply(500);
+
+      axiosInstance.post('http://tubular.azurewebsites.net/api/orders/paged', { fakeColumnStructure })
```

Comment:
The tests shouldn't be calling axiosInstance methods, please read carefully the docs.